frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -11,42 +11,65 @@ import {
 } from 'kubevirt-web-ui-components';
 import { ResourceSummary, NodeLink, ResourceLink } from '@console/internal/components/utils';
 import { PodKind } from '@console/internal/module/k8s';
-import { getName, getNamespace, DASH } from '@console/shared';
+import { getName, getNamespace } from '@console/shared';
 import { PodModel } from '@console/internal/models';
 import { VMKind, VMIKind } from '../../types';
 import { VMTemplateLink } from '../vm-templates/vm-template-link';
 import { getBasicID, prefixedID } from '../../utils';
 import { vmDescriptionModal, vmFlavorModal } from '../modals';
 import { getDescription } from '../../selectors/selectors';
 import { getVMStatus } from '../../statuses/vm/vm';
-import { FlavorText } from '../flavor-text';
+import { getFlavorText } from '../flavor-text';
 import { EditButton } from '../edit-button';
 import { getVmiIpAddressesString } from '../ip-addresses';
 
 import './_vm-resource.scss';
 
+export const VMDetailsItem: React.FC<VMDetailsItemProps> = ({
+  title,
+  idValue,
+  isNotAvail = false,
+  valueClassName,
+  children,
+}) => {
+  return (
+    <>
+      <dt>{title}</dt>
+      <dd id={idValue} className={valueClassName}>
+        {isNotAvail && <span className="text-secondary">Not available</span>}
```

Comment:
IMO its better to do
```suggestion
        {isNotAvail ? <span className="text-secondary">Not available</span> : children}
```

Reply:
done